### ISSUE TITLE
Disable bar based on a parameter/variable?

### ISSUE THREAD

**Issue by sortedcord** [rsalmei/alive-progress#103]
Is it possible to disable the bar based on certain conditions? A good example of what I'm saying could be something like:

```py
progress_bar = False
if progress_bar == False:
  # code to disable the progressbar
else:
  #progressbar remains enabled

with alive_bar(100) as bar:
  # Process code
```
In this case the visibility of alive_bar depends on the value of `progress_bar` ... Another solution that I thought of was writing the process code 2 times to account for the progress_bar value but that is quite unconventional...

**Comment by sortedcord** [comment]
Another way I tried to do this was something like so:

```py

def foo():
  try:
    br()
  except NameError:
    def br():
      pass
  
  #Some process code
  br()

if progress_bar == True:
  with alive_bar(1) as br:
    foo()
else:
  foo()
```
This is another hacky solution I came up with which works halfway... That is, if I use `.text` along with then it causes an error if br() is not defined. I cannot add a try exception case every time I use `br`

**Comment by TheTechRobo** [comment]
For future reference, you should use `if progress_bar` or `if progress_bar is True`, and `if not progressbar` or `if progress_bar is False`, not `==`. `is` is faster and better for this case, as `1 == True` is true but `1 is True` is false.

**Comment by rsalmei** [comment]
Yes, this is absolutely right, in this mode I disable every animated feature, actually I do not even start the background thread.
But the final receipt I do print, since I find it really important. It shows some processing did occur, and finished with success.
Is this a problem to show the final result only? Both @sortedcord (original OP) and @radugrosu.

**Comment by rsalmei** [comment]
Hey, the PR is ready to fix this! #111 
I'll release it very soon...

**Comment by rsalmei** [comment]
Hey, version `2.1` is released!!
This should be fixed, please report back if anything new arises.
